spec/lib/conjur/trusted_proxy_filter_spec.rb

```diff
@@ -2,12 +2,12 @@
 
```

Comment:
It looks like the `trusted_proxy_filter.rb` and `trusted_proxy_filter_spec.rb` files still need to be removed as part of this PR.